elements/pf-tabs/pf-tabs.ts

```diff
@@ -61,38 +65,150 @@ import styles from './pf-tabs.css';
  * @cssprop     {<color>} --pf-c-tabs__scroll-button--disabled--Color                 {@default `#d2d2d2`}
  */
 @customElement('pf-tabs')
-export class PfTabs extends BaseTabs {
-  static readonly styles = [...BaseTabs.styles, styles];
+export class PfTabs extends LitElement {
+  static readonly styles = [styles];
 
   protected static readonly scrollTimeoutDelay = 150;
 
-  static isTab(element: HTMLElement): element is PfTab {
-    return element instanceof PfTab;
-  }
-
-  static isPanel(element: HTMLElement): element is PfTabPanel {
-    return element instanceof PfTabPanel;
-  }
-
-  static isExpandEvent(event: Event): event is TabExpandEvent {
-    return event instanceof TabExpandEvent;
-  }
-
+  /**
+   * Box styling on tabs. Defaults to null
+   */
   @cascades('pf-tab', 'pf-tab-panel')
```

Comment:
If there's any way to get rid of these `@cascades` decorators (maybe a context?) it would help simplify things quite a bit IMO

can be a followup

Reply:
https://github.com/patternfly/patternfly-elements/issues/2651